src/olympia/addons/models.py

```diff
@@ -1540,7 +1540,7 @@ def check_and_update_sensitive_permissions(self):
 
         version = self.versions.latest()
 
-        if not version or not version.all_files[0]:
+        if not version or len(version.all_files) == 0:
```

Comment:
Versions should have files, but if they don't make sure they don't just error out like this use to.